lsp/lsp-harness/src/lib.rs

```diff
@@ -92,6 +93,28 @@ impl Parse for Request {
                 work_done_progress_params: Default::default(),
                 partial_result_params: Default::default(),
             })),
+            // The format of a Completion request is the file:line:char location,
```

Comment:
It's not too important for now, but I wonder if we should follow the same approach as for the integration tests in the future, where we simply write TOML inside comments - might make both using the format and parsing it uniform and less idiosyncratic.